spec/fabricators/status_reaction_fabricator.rb

```diff
@@ -0,0 +1,6 @@
+Fabricator(:status_reaction) do
+  account      nil
+  status       nil
+  name         "MyString"
+  custom_emoji nil
+end
```

Comment:
```suggestion
# frozen_string_literal: true

Fabricator(:status_reaction) do
  account
  status
  name         '👍'
  custom_emoji nil
end
```

At least making the fabricator usable.

Reply:
Hmh this seems to cause a failure:
```
  1) The status_reaction factory is valid
     Failure/Error: expect(factory).to be_valid
       expected #<StatusReaction id: 1, account_id: 110339817053204859, status_id: 110339817053191929, name: "👍", custom_emoji_id: nil, created_at: "2023-05-09 17:06:33.686302000 +0000", updated_at: "2023-05-09 17:06:33.686302000 +0000"> to be valid, but got errors: Limit of different reactions reached
     # ./spec/fabricators_spec.rb:9:in `block (3 levels) in <top (required)>'
```

Reply:
Hm, that's odd indeed. I'll not apply this for now.

Reply:
Yeah, I don't know how to fix that.
The fabricator itself works, it's just the fabricators_spec which fails